README.md

```diff
@@ -3,18 +3,34 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/MysteriumNetwork/node)](https://goreportcard.com/report/github.com/MysteriumNetwork/node)
 [![Build Status](https://travis-ci.org/MysteriumNetwork/node.svg?branch=master)](https://travis-ci.org/MysteriumNetwork/node)
 
-VPN server and client for Mysterium Network
+Cross-platform software to run a node in Mysterium Network. It contains Mysterium server (node), 
+client API (tequila API) and client-cli (console client) for Mysterium Network.
+ 
+Currently node supports OpenVPN as its underlying VPN transport. 
 
-- Homepage https://mysterium.network/
+## Getting Started
+
+- Homepage https://mysterium.network
```

Comment:
Did you make this not as `[Homepage](link)`, not like others intentionally or accidentally?

Reply:
Its intentional. We need not just a link to our homepage, bet we want it to be memorised.

Reply:
You could do [Homepage https://mysterium.network](https://mysterium.network)